packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -503,10 +503,11 @@ def _download_bundle_from_master(dcos_api_session, master_index):
     bundles = _get_bundle_list(dcos_api_session)
     assert bundles
 
-    expected_common_files = ['dmesg-0.output.gz', 'opt/mesosphere/active.buildinfo.full.json.gz',
+    expected_common_files = ['dmesg_-T-0.output.gz', 'opt/mesosphere/active.buildinfo.full.json.gz',
                              'opt/mesosphere/etc/dcos-version.json.gz', 'opt/mesosphere/etc/expanded.config.json.gz',
                              'opt/mesosphere/etc/user.config.yaml.gz', 'dcos-diagnostics-health.json',
-                             'var/lib/dcos/cluster-id.gz']
+                             'var/lib/dcos/cluster-id.gz', 'timedatectl-5.output.gz',
```

Comment:
This is introducing some flakiness into the system.

The failure here in Teamcity https://teamcity.mesosphere.io/viewLog.html?buildId=1192528&buildTypeId=DcOs_Enterprise_Test_DockerBased_DockerWStaticBackendAndSecurityStrict 

is related to not finding `'timedatectl-5.output.gz'`

Reply:
Looks like we missed it during the review. What's the reason for `'timedatectl-5.output.gz'` in the expected files. Especially the `-5` part in the file name.  The master PR has a different name https://github.com/dcos/dcos/pull/3148/files 

And shouldn't the assertion be more generic? Look only for the filename instead of the `-number` too which seems to vary?